package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xpring-common-js",
-  "version": "2.0.1",
+  "version": "2.1.0",
```

Comment:
Since we increment the version number here, we should add a new `## [2.1.0]` header in the CHANGELOG.

(Or potentially a `## [3.0.0]` header depending on what Keefer says about the breaking change).